components/Tabs/src/Theming/MaterialTabs+Theming.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Update the copyright notice to 2019 please.

Reply:
Addressed in 82f41bedaeaae0376513d405303d6d8b4f320c34.